modules/apps/content-dashboard/content-dashboard-test/src/testIntegration/java/com/liferay/content/dashboard/web/internal/portlet/test/ContentDashboardAdminPortletTest.java

```diff
@@ -237,6 +237,39 @@ public void testGetAuditGraphTitleWithMissingChildAssetVocabulary()
 		}
 	}
 
+	@Test
+	public void testGetContextWithLtrLanguageDirection() throws Exception {
+		MockLiferayPortletRenderRequest mockLiferayPortletRenderRequest =
+			_getMockLiferayPortletRenderRequest();
+
+		mockLiferayPortletRenderRequest.setAttribute(
+			WebKeys.COMPANY_ID, _company.getCompanyId());
+
+		Map<String, Object> data = _getData(mockLiferayPortletRenderRequest);
+
+		Map<String, Object> context = (Map<String, Object>)data.get("context");
+
+		Assert.assertNotNull(context);
+		Assert.assertFalse((Boolean)context.get("rtl"));
```

Comment:
Maybe it is better to read "dir" instead of "RTL"

Reply:
yes, as I said it makes sense for me :)

Reply:
Finally, I'm using "languageDirection" as it's been already used in several parts